notebooks/tel_and_site/subsys_req_ver/m1m3/SITCOM-1136_M1M3_analysis_stability_tracking.ipynb

```diff
@@ -0,0 +1,704 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #1.    unstable_events = {}
</pre><p> </p><p>Same thing here, specially because this is a big cell with several steps. Put it inside a subsection explaining its purpose might be helpful to follow the results you obtain.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/lsst-sitcom/notebooks_vandv/pull/107/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='21'/>

Reply:
<p>I have added some comments</p>

Reply:
<p>Same thing here. I don't see the commments.</p>

Reply:
<p>I have added a box in markdown with a short explanation</p>